### PR TITLE
tiny fixes

### PR DESCRIPTION
**Pull request by pitis91**
added temple of equilibrium vines script (just missing to add my map)
fixed ruthless seven corpse bosses
added transcripts to nina
fixed servant sentry help words (missing to add timer to each 24 o 20h return to give the slime gobbler)
removed message from ungreez creaturescript
removed new frontier secured exit from minotaur
removed pits of inferno message on each shortcut

### CODE REVIEW

File: data/actions/actions.xml [hunk 1]
```diff
@@ -42,6 +42,9 @@
 	<!-- Farmine -->
 	<action fromaid="9118" toaid="9121" script="farmine/elevatorLever.lua" />
 
+	<!-- Temple of Equilibrium -->
```
Comment (pitis91) [inline data/actions/actions.xml]: I don't know how to create a folder can someone help me?

Comment (EPuncker) [inline data/actions/actions.xml]: just add the folder normally in windows and in git shell you write:

git add

and press "tab" until the right folder appears and press enter

Comment (pitis91) [inline data/actions/actions.xml]: I don't know how to use git shell I wasted some time to learn it but nothing happends >.<

**Comment by pitis91** [comment]
thanks a lot @EPuncker now I can create folders, and pulls of more than one file =D

**Comment by pitis91** [comment]
and now what's up?? :/

**Comment by ninjalulz** [comment]
It's still quite a lot of commits, you should be able to push multiple changes at once.

**Comment by EPuncker** [comment]
something like:

git reset --soft HEAD~15
git commit
git push -f

should do the trick (it did with me last time lol)

**Comment by pitis91** [comment]
thanks @EPuncker another time for your help :D

**Comment by EPuncker** [comment]
now just edit the commit message to have all info

**Comment by pitis91** [comment]
what command is it?

**Comment by EPuncker** [comment]
here in this link were we are talkin, just click the "pencil" thing and add the message

**Comment by pitis91** [comment]
done?

**Comment by EPuncker** [comment]
yeah :D

**Comment by pitis91** [comment]
another time... thankkkss!

**Comment by pitis91** [comment]
I think the new Pull is more clean, and I've just practised how to create pull etc etc, "improving my skills" for future works.

**Pull request closed without merging**